src/test/suite/views/webviewController.test.ts

```diff
@@ -849,4 +849,55 @@ suite('Webview Test Suite', () => {
       );
     });
   });
+
+  test('webview returns the feature flags on a feature flags request', (done) => {
+    const extensionContextStub = new ExtensionContextStub();
+    const testStorageController = new StorageController(extensionContextStub);
+    const testTelemetryService = new TelemetryService(
+      testStorageController,
+      extensionContextStub
+    );
+    const testConnectionController = new ConnectionController({
+      statusView: new StatusView(extensionContextStub),
+      storageController: testStorageController,
+      telemetryService: testTelemetryService,
+    });
+    let messageReceived;
+
+    const fakeWebview = {
+      html: '',
+      postMessage: (message): void => {
+        assert(message.command === 'FEATURE_FLAGS_RESULTS');
+        assert(typeof message.featureFlags === 'object');
```

Comment:
```suggestion
        assert.strictEqual(message.command, 'FEATURE_FLAGS_RESULTS');
        assert.strictEqual(typeof message.featureFlags, 'object');
```

Reply:
I'm thinking we should add a lint rule for this 😅 . Or start using chai everywhere in the extension. I'm kind of leaning towards chai.